t/10-tests_overview.t

```diff
@@ -229,14 +229,23 @@ $t->element_exists('#res_DVD_i586_kde',                           'job from grou
 $t->element_exists('#res_GNOME-Live_i686_RAID0 .state_cancelled', 'another job from group 1001');
 $t->element_exists('#res_NET_x86_64_kde .state_running',          'job from group 1002 is shown');
 
+# $t->get_ok('/tests/overview?distri=opensuse&version=13.1&groupid=1001&groupid=1002')->status_is(200);
+# $summary = get_summary;
+# like(
+#     $summary,
+#     qr/Summary of opensuse, opensuse test build 0091 /i,
+#     'multiple groups with no build specified yield latest build of every group'
+# );
+# like($summary, qr/Passed: 2 Failed: 0 Scheduled: 1 Running: 2 None: 1/i);
+
 $t->get_ok('/tests/overview?distri=opensuse&version=13.1&groupid=1001&groupid=1002')->status_is(200);
```

Comment:
Instead of adding the job to the lengthy fixture list breaking many other test modules better just create a single job before you do the "get", e.g.

```perl
$t->app->schema->resultset('Jobs')->create({
        id => 99964,
        BUILD => '0092',
        group_id => 1002,
        TEST => 'kde'
    });
```